static/css/styles.css

```diff
@@ -100,21 +101,57 @@ input {
     padding: 10px 30px;
 }
 
-.primary-content {
-    float: left;
+.helping-text {
+    margin: 0 5px 5px 0;
```

Comment:
The helping-text, when empty, is pushing everything down by 5px so the vertical centering is off. I guess we should hide and show it as necessary in addition to changing the text (or just remove the vertical margin, which is also adding unnecessary height in the student view).

Also, a side effect of moving helping-text out of the buttons container is that the name/andrewID/question text is also getting shifted down when the helping-text has content (see screenshot). Not a huge deal in my opinion and doesn't seem too easy to fix, so I think it's fine for now.

<img width="1013" alt="Screen Shot 2020-01-19 at 11 42 16 AM" src="https://user-images.githubusercontent.com/1310713/72687235-c1e6b980-3ab0-11ea-95b3-af215b065ccf.png">

Reply:
Surprisingly the :not and :empty pseudoselectors are well-supported on browsers except IE, so I just made the CSS conditional on .helping-text being non-empty.

For your latter comment - I did that intentionally. The container height is going to increase regardless, so I thought it best to keep the alignment of text with buttons consistent.

Reply:
Okay, sounds good